site/role/manifests/gitlab.pp

```diff
@@ -0,0 +1,5 @@
+class role::gitlab {
+
+  include profile::gitlab
```

Comment:
add profile::base here, since other roles say all roles should include profile::base, perhaps?